packages/keyring-controller/src/KeyringController.test.ts

```diff
@@ -527,18 +517,23 @@ describe('KeyringController', () => {
           });
 
           it('should set default state', async () => {
-            await withController(async ({ controller }) => {
-              expect(controller.state.keyrings).not.toStrictEqual([]);
-              const keyring = controller.state.keyrings[0];
-              expect(keyring.accounts).not.toStrictEqual([]);
-              expect(keyring.type).toBe('HD Key Tree');
-              expect(controller.state.vault).toBeDefined();
-            });
+            await withController(
+              { cacheEncryptionKey, skipVaultCreation: true },
+              async ({ controller }) => {
+                await controller.createNewVaultAndKeychain(password);
```

Comment:
This test was relying upon the helper function calling the code under test. This made it difficult to understand. It has been updated to move the code-under-test directly into the test, so it's easier to tell what the test is doing.